indigo/distribution.yaml

```diff
@@ -780,6 +780,14 @@ repositories:
       url: https://github.com/kejriwalnishant1990/bowpmap_ros.git
       version: master
     status: maintained
+  bowpmap_ros-release:
```

Comment:
Please trust us that you do not want the same repository for your source and release repo. The release repos following the git buildpackage layout get large quickly in ways that are not conducive to source checkouts.